posthog/settings/ingestion.py

```diff
@@ -15,6 +15,7 @@
 EVENT_PARTITION_KEYS_TO_OVERRIDE = get_list(os.getenv("EVENT_PARTITION_KEYS_TO_OVERRIDE", ""))
 
 LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS = get_list(os.getenv("LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS", ""))
+LIGHTWEIGHT_CAPTURE_ENDPOINT_ALL = "*" in LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS
```

Comment:
Let's do this to be safe - some tokens could include `*` couldn't they?
```suggestion
LIGHTWEIGHT_CAPTURE_ENDPOINT_ALL = "*" == LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS
```

Reply:
`LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS` (declared on the line above that) is a list, already split on the comas. We're checking that one of the items in that list is `"*"`. This way we're less finicky than a strict `envvar === "*"` check.